tap_asana/streams/users.py

```diff
@@ -1,27 +1,21 @@
-
 from tap_asana.context import Context
 from tap_asana.streams.base import Stream
 
 
 class Users(Stream):
-  replication_method = 'FULL_TABLE'
-  name = 'users'
-
-  fields = [
-    "gid",
-    "resource_type",
-    "name",
-    "email",
-    "photo",
-    "workspaces"
-  ]
+    replication_method = "FULL_TABLE"
+    name = "users"
 
+    fields = ["gid", "resource_type", "name", "email", "photo", "workspaces"]
```

Comment:
Be consistent about the the way `fields` defined as you have done it in teams.py